### DISCUSSION TITLE
Dot net problem

### DISCUSSION THREAD

**Discussion by edris6** [SteamRE/DepotDownloader#348] [Q&A]
So im currently trying to downgrade human fall flat running this command: .\DepotDownloader.exe -app 477160 -depot 477161 -manifest 5187084049243380275

And i keep getting this error:
--------------------------------------------------------------------------------------------------------------------------------------------

A fatal error occurred. The required library hostfxr.dll could not be found.

If this is a self-contained application, that library should exist in [C:\Users\socce\Downloads\depotdownloader-2.4.6\].

If this is a framework-dependent application, install the runtime in the global location [C:\Program Files\dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime location in [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x64\InstallLocation].

The .NET runtime can be found at:
  - https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win10-x64&apphost_version=6.0.4

--------------------------------------------------------------------------------------------------------------------------------------------

I looked at past issues, to see if i can fix it, i have dot net installed, and i dont know what to do.

Please help

**Comment by yaakov-h** [comment]
It seems like you might have an older version of the .NET Runtime, but not .NET 6.

Can you please share the output of `dotnet --info`?

**Comment by edris6** [comment]
![imagem](https://user-images.githubusercontent.com/78484256/184531569-569739f1-ba4a-4d3a-84c4-477a6e5c282c.png)
Even doe i do have dotnet installed, the command  doesnt work
![imagem](https://user-images.githubusercontent.com/78484256/184531625-c14d5310-9653-431d-b7b3-2441467b8921.png)

**Reply by yaakov-h** [reply]
You're missing the .NET runtime entirely. Please follow the instructions in the error message that you pasted to download and install the .NET 6 runtime.

**Comment by edris6** [comment]
Console, or desktop
